rest-server/src/main/resources/nubesgen/.github/workflows/gitops.yml.mustache

```diff
@@ -86,6 +86,44 @@ jobs:
           application_name: ${{ steps.infrastructure-deployment.outputs.application_name }}
           container_registry_name:  ${{ steps.infrastructure-deployment.outputs.container_registry_name }}
     <%/runtimeQuarkus%>
```

Comment:
For the docker micronaut variants, I put the steps here instead of creating actions as with the other runtimes

If I had done PRs for actions, it would have been impossible to functionally test